src/third-party-tags/inizio.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { inizio } from './inizio';
+
+describe('index', () => {
+	it('should use the feature switch option', () => {
+		const inizioInstance = inizio({ shouldRun: true });
+		expect(inizioInstance).toStrictEqual({
+			shouldRun: true,
+			url:
+				'//cdn.brandmetrics.com/survey/script/e96d04c832084488a841a06b49b8fb2d.js',
+			name: 'inizio',
+			onLoad: inizioInstance.onLoad,
```

Comment:
is this testing anything?

Reply:
Not really…

Reply:
ah i guess it's needed for the `toStrictEqual`?

Reply:
Actually, it makes more sense to use `toMatchObject` in this context, as in #52.